sklearn/tests/test_pipeline.py

```diff
@@ -1071,3 +1071,13 @@ def test_make_pipeline_memory():
     assert pipeline.memory is None
 
```

Comment:
you can add `assert len(pipeline) == 2` here and remove the test you've added bellow.

Reply:
ok changing.